components/payments/internal/app/storage/payments.go

```diff
@@ -180,10 +180,10 @@ func (s *Storage) UpsertPayments(ctx context.Context, provider models.ConnectorP
 		}
 	}
 
-	err = s.UpdateTransfersFromPayments(ctx, payments)
-	if err != nil {
-		return e("failed to update transfers", err)
-	}
+	//err = s.UpdateTransfersFromPayments(ctx, payments)
```

Comment:
Do we want to keep that commented?

Reply:
Just a quick hack for @reslene , this PR must not be merged :)